apps/site/src/lib/components/HackLogo/HackLogo.tsx

```diff
@@ -0,0 +1,19 @@
+import hackAnteater from "@/assets/logos/hackuci-anteater.svg";
+import hackGear from "@/assets/logos/hackuci-gear.svg";
+import Image from "next/image";
+import styles from "./HackLogo.module.scss";
```

Comment:
Nitpick [repeating]: We usually order imports by (from top to bottom) first party libraries (ex. Next/React related imports), third party libraries, TS(X) file imports, images, and then SCSS files.